test/functional/apps/discover/_runtime_fields_editor.ts

```diff
@@ -44,18 +44,20 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     after(async () => {
-      await kibanaServer.uiSettings.replace({ 'discover:searchFieldsFromSource': true });
+      await esArchiver.unload('logstash_functional');
```

Comment:
don't think this needs to be unloaded since explicitly since this is done in the file that includes `_runtime_fields_editor.ts`

https://github.com/elastic/kibana/blob/2d0b32a40afc2e095b035d27cfc95c5e5f6c74b2/test/functional/apps/discover/index.ts#L20-L25